modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -61,7 +61,7 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 +
 [NOTE]
 ====
-If you do not control the DNS server, you can add the record to your `/etc/hosts` file. This action makes the API accessible to only you, which is not suitable for production deployment but does allow installation for development and testing.
+If you do not control the DNS server, you can add the records such as web console, OAuth, Prometheus and Grafana to your `/etc/hosts` file. The `/etc/hosts` file does not consider wildcard entries. You need a list of individual URLs for your applications in the `/etc/hosts` file. This action makes the API and applications accessible to only you, which is not suitable for production deployment, but does allow installation for development and testing.
```

Comment:
@xenolinux,

So I might word this differently, because no one probably knows what the actual _DNS name_ is for these API endpoints. For example, they must be explicitly:

```
$APPS_FLOATING_IP console-openshift-console.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
$APPS_FLOATING_IP integrated-oauth-server-openshift-authentication.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
$APPS_FLOATING_IP oauth-openshift.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
$APPS_FLOATING_IP prometheus-k8s-openshift-monitoring.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
$APPS_FLOATING_IP grafana-openshift-monitoring.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
```

So we should be explicit rather than just say _Grafana_ for example without stating what exactly the _DNS name_ is for this, which is `grafana-openshift-monitoring`. So this could probably be:

> If you do not control the DNS server, you can add the records for services such as the following to your `/etc/hosts` file:
> * Grafana: `grafana-openshift-monitoring`
> * ect.
> The `/etc/hosts` file does... [rest of the paragraph]
> [rest of text from above...]

In fact, I _think_ you'd be best to specify the entire line for the hosts file entry, which would be:

`oauth-openshift.apps.<CLUSTER_NAME>.<BASE_DOMAIN> <API_<FIP>`

So a user might have:

```
oauth-openshift.apps.example.com 10.10.1.1
```

And then repeat for each of these. That will provide a working configuration.

Reply:
good point. Just a note: in `/etc/hosts`, you need the IP first, and then the name.

```plaintext
10.10.1.1 oauth-openshift.apps.example.com
```

Reply:
Addressed- @jboxman-rh @pierreprinetti Could you please take a look if it looks good now? I mentioned `<APPLICATION_FLOATING_IP>` in the example.